### DISCUSSION TITLE
Model Gorilla Rack Shelving - 77"W x 24"D in NetBox?

### DISCUSSION THREAD

**Discussion by basictheprogram** [netbox-community/netbox#9753] [Q&A]
Just started using NetBox. We have several Gorilla Rack Shelving (image attached) that hold Lenovo 30BH0029US (image attached). I've been adding them to the Site as "Non-Racked Devices".  Worked ok for 1 shelf, but we have 3 of these shelves filled with Lenovo 30BH0029US. Is there a better way to track this info in NetBox, other than "Non-Racked Devices"?

Thanks.

<img width="220" alt="Gorilla_Rack_Shelving_-_77_W_x_24_D_-_Sam_s_Club" src="https://user-images.githubusercontent.com/6723257/179403391-053de1dd-a325-441d-924b-edacea032092.png">

<img width="106" alt="Cursor_and_Amazon_com__Lenovo_SY_30BH0029US_ThinkStation_P320_i7-7700_16GB_512GB_SSD_W10P_Retail___Electronics" src="https://user-images.githubusercontent.com/6723257/179403447-23c59c29-609f-4934-95c8-40876d802b59.png">

**Comment by candlerb** [comment]
The "racks" in Netbox are traditional 2/4 post 19" (or similar) racks.

However, you can create nested "locations" within a site. AFAIK, if your version of Netbox is reasonably recent, a device can be non-racked but associated with a location.  Hence you could create a location representing the shelving system, and separate locations within it for each shelf.

Give that a try anyway.
